lib/game.rb

```diff
@@ -0,0 +1,39 @@
+require "csv"
+
+class Game
+  @@all = nil
+
+  def self.all
+    @@all
+  end
+
+  def self.from_csv(csv_file_path)
+    csv = CSV.read("#{csv_file_path}", headers: true, header_converters: :symbol)
+    @@all = csv.map { |row| Game.new(row) }
```

Comment:
I updated this code to mimic Ross' one-line syntax preference so that our classes would be totally synchronized which will hopefully streamline the inheritance refactoring